docs/faq/migration/asset-migration.md

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_label: 'Asset Migration'
+sidebar_position: 2
+---
+
+# Asset Migration
+
+### Will previous Universal Profiles be migrated to mainnet?
+
+LUKSO will migrate all Universal Profiles previously created on the L14 Testnet by [The Dematerialised Marketplace](https://thedematerialised.com/) or [Universal Profile Explorer](https://universalprofile.cloud/). If you deployed custom profiles on older testnets, you would be responsible for migrating them to the mainnet.
```

Comment:
there are more we will migrate, i dont have the list here but there are more.

Reply:
There are more, but they were all deployed on L14 profile explorer no? Is anyone to ask for this? I just took the question from the old FAQ board. @Hugoo

Reply:
Ok then let's keep it like this for now ;)